manifests/stage-rdma-device-plugin/0040_scc.openshift.yaml

```diff
@@ -0,0 +1,47 @@
+# Please edit the object below. Lines beginning with a '#' will be ignored,
```

Comment:
what will happen if you try to deploy this on vanilla k8s ?
it does not have `security.openshift.io/v1/SecurityContextConstraints`

Reply:
I don't know, I've only tested this on OpenShift. Presumably it wouldn't create this object?

Reply:
i think it would fail to create it, lets wait for CI results.

Reply:
yep, fails on vanilla k8s.

```
 no matches for kind \"SecurityContextConstraints\" in version \"security.openshift.io/v1\""
```

@AbdYsn  Thanks for posting the error !

Reply:
@adrianchiris I've settled on an efficient way to prevent `SecurityContextConstraints` from being created on OpenShift by making them conditional on `.RuntimeSpec.OSName`. This way there are no code changes necessary and any OpenShift objects can just follow this pattern. What do you think?

Reply:
Does openshift run only on `rhcos` ? i.e no other OSes that support Openshift e.g RHEL ?
And the other way around:
Does `rhcos` only run openshift ? (i.e no vanilla k8s runs on it?)

I know its not a problem now, but i wonder if this assumption will get us into trouble later down the road.

Reply:
> Does OpenShift only run on `rhcos`?
Yes, OpenShift is only supported on RHCOS. We used to support RHEL7 worker nodes, but no longer.

> Does `rhcos` only run OpenShift?
Yes.

Reply:
OK i understand. So we can go with that IMO.